ruby/private/gem.bzl

```
@@ -13,10 +13,15 @@ def rb_gem(name, version, gem_name, srcs = [], **kwargs):
     source_date_epoch = kwargs.pop("source_date_epoch", None)
     verbose = kwargs.pop("verbose", False)
 
+    strip_paths = []
+    if "strip_paths" in kwargs and "require_paths" not in kwargs:
+        fail("Must specify 'require_paths' when using the 'strip_paths' argument")
```

```suggestion
    if strip_paths and "require_paths" not in kwargs:
        fail("Must specify 'require_paths' when using the 'strip_paths' argument")
```

Also add `strip_paths = []` to the parameter list of `rb_gem`.